pkgs/development/tools/continuous-integration/concourse-bin/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib
+, fetchurl, autoPatchelfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "concourse-bin";
+  version = "6.5.1";
+
+  src = fetchurl {
+    url = "https://github.com/concourse/concourse/releases/download/v${version}/concourse-${version}-linux-amd64.tgz";
+    sha256 = "ec0ed4a68a7221edea8ded0f569655298419ef4de8f9193b59dcd5098a4a360c";
+  };
+
+  phases = [ "unpackPhase" "installPhase" "fixupPhase" ];
```

Comment:
I don't think you need to explicitly list the phases like this - I'm guessing you're intending to exclude buildPhase? Is there a Makefile in this release tarball? If there's no Makefile then the buildPhase won't do anything

Reply:
Right . Fixed.